.github/workflows/build_and_test.yml

```
@@ -183,6 +183,11 @@ jobs:
           cd python/test/unit
           python3 -m pytest -n 8 --verbose operators
 
+      - name: Run partial regression tests
+        run: |
+          cd python/test
+          python3 -m pytest -n 8 --verbose regression/ --ignore=regression/test_performance.py --ignore=regression/test_functional_regressions.py
```

To match integreation-tests.yml
```suggestion
      - name: Regression tests
        run: |
          cd python/test/regression
          python3 -m pytest -vvv -s . --rerun 10 --ignore=test_performance.py --ignore=test_functional_regressions.py
```

---

Fixed